Microsoft.AVS.VVOLS/Microsoft.AVS.VVOLS.psd1

```diff
@@ -0,0 +1,136 @@
+#
+# Module manifest for module 'Microsoft.AVS.VMFS'
+#
+
+@{
+
+    # Script module or binary module file associated with this manifest.
+    RootModule        = 'Microsoft.AVS.VVOLS.psm1'
+
+    # Version number of this module.
+    ModuleVersion     = '1.0.0'
+
+    # Supported PSEditions
+    # CompatiblePSEditions = @()
+
+    # ID used to uniquely identify this module
+    GUID              = '5fa1a15b-4953-4eba-9d69-1cf8d7a94f70'
+
+    # Author of this module
+    Author            = ''
+
+    # Company or vendor of this module
+    CompanyName       = ''
+
+    # Copyright statement for this module
+    Copyright         = ''
+
+    # Description of the functionality provided by this module
+    Description       = 'Azure VMware Solutions VMFS Package (Automation only).'
+
+    # Minimum version of the PowerShell engine required by this module
+    PowerShellVersion = '5.1'
+
+    # Name of the PowerShell host required by this module
+    # PowerShellHostName = ''
+
+    # Minimum version of the PowerShell host required by this module
+    # PowerShellHostVersion = ''
+
+    # Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # DotNetFrameworkVersion = ''
+
+    # Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # ClrVersion = ''
+
+    # Processor architecture (None, X86, Amd64) required by this module
+    # ProcessorArchitecture = ''
+
+    # Modules that must be imported into the global environment prior to importing this module
+    RequiredModules   = @(
+        @{ "ModuleName" = "Microsoft.AVS.Management"; "ModuleVersion" = "5.2.39" }
```

Comment:
We're on https://www.powershellgallery.com/packages/Microsoft.AVS.Management/5.3.88 at the moment, let's have that version as the dependency.

Reply:
Updated